src/GitVersion.Core/Configuration/GitVersionConfiguration.cs

```diff
@@ -14,6 +14,9 @@ public GitVersionConfiguration()
         Ignore = new IgnoreConfiguration();
     }
 
+    [YamlMember(Alias = "remote-name-in-git")]
+    public string? RemoteNameInGit { get; set; }
```

Comment:
isn't `DefaultRemoteName` a better name to be used?

Reply:
I have reverted this settings because it's problematic to restrict the user on one remote in GIT. I would say to remove or rewrite the test who is failing.